src/vic.h

```diff
@@ -77,6 +77,9 @@
 extern char vic_buffer[VIC_BUFFER_SIZE + 1];
 extern uint8_t vic_buffer_len;
 
+#define vic_printf(format, ...) do { char _vic_buff[VIC_BUFFER_SIZE];\
```

Comment:
Thanks @burlito and @vargac.

The way I read this is that we can go forward with it and give it a try. I am pretty sure that @vargac will come back and reiterate the way this is currently done for both input and output (a centralized way of doing that might indeed be the answer here).

Also, we should certainly take a look at `xitoa` once we do that. How we will include it in the general Arduino pipeline is still unclear to me but I guess that is left for me to figure out...